lib/workers/repository/updates/__snapshots__/generate.spec.ts.snap

```diff
@@ -102,7 +102,7 @@ exports[`workers/repository/updates/generate generateBranchConfig() handles @typ
   "prBodyColumns": [],
   "prTitle": "some-title",
   "prettyDepType": "dependency",
-  "recreateClosed": false,
+  "recreateClosed": true,
```

Comment:
Can this test result change be fixed?

Reply:
yes, since the old logic didn't allow a immortal pr new shouldn't either. I guess it was a wrong change to begin with from my side. I should have used 'never' instead of 'auto'